README.md

```diff
@@ -28,23 +28,69 @@ See the official [React Native website](https://facebook.github.io/react-native/
+	- You cannot run the emulators and some other developer tooling on the "Starter" or "Home" versions of these operating systems
+	- You can run React Native Windows UWP apps only on Windows 10 devices, but React Native Windows WPF apps will run on Windows 7-10 so long as .NET 4.6 is installed on the end user's machine
 - Download [Visual Studio 2017 Community or Greater](https://www.visualstudio.com/downloads/). (Visual Studio 2015 support is being deprecated.)
-- - You will need to start Visual Studio after it is installed to do some final setup before it can be used to build or run your React Native Windows application
+	- You will need to start Visual Studio after it is installed to do some final setup before it can be used to build or run your React Native Windows application
 - [Windows 10 SDK Build 14393](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive)
 
 *Note*: Development on React Native Windows itself currently requires Visual Studio 2017. It is not supported with VS Code, but we will gladly accept pull requests to enable a great developer experience in those environments.
 
 ## Getting Started
 
-- Follow the [Getting Started guide](http://facebook.github.io/react-native/docs/getting-started.html) to install React Native and its dependencies.
-- Change your working directory to the project directory you specified in the `react-native init` command from the [Getting Started guide](http://facebook.github.io/react-native/docs/getting-started.html) (e.g., `cd AwesomeProject`).
-- Install [rnpm-plugin-windows](https://www.npmjs.com/package/rnpm-plugin-windows) from NPM (i.e., `npm install --save-dev rnpm-plugin-windows` or `yarn add rnpm-plugin-windows --dev` if you have Yarn installed).
-- Initialize Windows project with `react-native windows` command in the project directory.
-- *Note*: You will need to install the [requirements](#system-requirements) before you can build and run your application.
+### Installing React Native and its dependencies
+This is a targeted summary of setup React Native setup for specifically React Native Windows. See the [React Native Getting Started Guide](http://facebook.github.io/react-native/docs/getting-started.html) for full details.
+
+#### Dependencies: Node, Python, JDK
+Install the dependencies [specified by React Native](http://facebook.github.io/react-native/docs/getting-started.html#node-python2-jdk). Specifically, make sure a recent version of Node, Python2, and a recent version of the Java SE Development Kit (JDK) are installed.
+
+[Chocolatey](https://chocolatey.org/) is the React Native recommended installation method. On an elevated Command Prompt, run:
+```
+choco install -y nodejs.install python2 jdk8
+```
+
+#### React Native CLI
+Next, we need the React Native command line interface. Install using NPM:
+```
+npm install -g react-native-cli
+```
+
+#### Initialize your project
```

Comment:
We may want to include a note that this is not yet compatible with create-react-native-app because Expo does not yet support react-native-windows. We can point to instructions on how to eject the project (without Expo) so users can add react-native-windows.